packages/wallets/src/App.tsx

```diff
@@ -7,7 +7,7 @@ import './index.scss';
 import './translations/i18n';
 
 const App: React.FC = () => (
-    <APIProvider standalone>
+    <APIProvider customLoginIDKey='active_wallet_loginid' standalone>
```

Comment:
naming: just "loginIdKey"? :D no need for "custom" as every parameter is "custom" really